modules/learn/pages/data/change-history.adoc

```diff
@@ -0,0 +1,245 @@
+= Change History
+:description: pass:q[When Magma storage is used for a bucket, the changes made to documents within the bucket's collections can be recorded, in a _change history_.]
+
+[abstract]
+{description}
+
+[#understanding-change-history]
+== Understanding Change History
+
+When xref:learn:buckets-memory-and-storage/storage-engines.adoc#storage-engine-magma[Magma] storage is used for a bucket, the changes made to documents within the bucket's collections can be recorded, in a _change history_.
+The change history resides on disk.
+Its capacity is administrator-specified.
+When the change history is full, old records are automatically removed (by means of compaction), to allow space for new records.
+
+Change history can be either _on_ or _off_.
+This is determined at bucket-level: if the change history is _on_, it is potentially active for _all collections_ in the bucket.
+
+If change history is _on_, the size of the change history should be specified in both _seconds_ and _bytes_:
```

Comment:
Suggest redraft this sentence - the "If change history is on" contradicts the "should"  later in the sentence?

Maybe something like "The size of the change history can be specified.... When either or both are configured the change history is _on_" ?

Reply:
Although later in the section you make the distinction about "either" config enables the feaure

Reply:
I've removed the later sentence, and here, have written: The size of the change history can be specified both in _seconds_ and in _bytes_.
If either or both are configured, change history is _on_.